src/inc/common/Mlp_Helpers.php

```diff
@@ -479,7 +479,7 @@ public static function show_linked_elements( $args ) {
 
 		$output .= '</ul></div>';
 
-		return $output;
+		return apply_filters( 'show_linked_elements', $output, $items, $params );
```

Comment:
> In my opinon `mlp_linked_elements_html` would better fit the existing [filters](../search?utf8=✓&q=apply_filters).

That's exactly what it will be. :+1:

We don't name filter hooks just like the function/method they live in, but choose more sensible names. `show_linked_elements` (prefixed or not) may be misinterpreted as some sort of boolean switch for some functionality.
So `mlp_linked_elements_html` it is.

Also, for the sake of easier debugging, the result of the filter should be cached in the `$output` variable, which is returned the line **after** that. This way, one can step through the code, and easily see the original output and the filtered one.

Lastly, the filter is lacking appropriate documentation. Please [see this one](https://github.com/inpsyde/multilingual-press/blob/29bc98a1dca2e71c813150a62ca898d406a1ce3f/src/Common/Type/FilterableTranslation.php#L154-L171) as an example.

@dasllama would you like to take care of that, or do you just want this included? :)